Backpack/src/androidTest/java/net/skyscanner/backpack/util/BpkThemeTest.kt

```diff
@@ -67,7 +67,7 @@ class BpkThemeTest {
 
     BpkTheme.applyDefaultsToContext(testContext)
     testContext.theme.resolveAttribute(R.attr.bpkPrimaryColor, t, true)
-    Assert.assertEquals(blue500, t.data)
+    Assert.assertEquals(skyBlue, t.data)
```

Comment:
BpkTheme.getPrimaryColor docs refer to the wrong colour now.

Reply:
done